source/includes/api/_authentication.md

```diff
@@ -32,3 +32,13 @@ curl -X POST https://api.resourcewatch.org/auth/user \
 ```
 
 There are three allowed roles: `USER`, `MANAGER` and `ADMIN`. The 'apps' field only permits applications that are powered by the API: `rw`, `gfw`, `prep`, etc.
+
+## Troubleshooting auth problems
+
+If you are having trouble with authentication, please make sure that you have followed the steps [detailed here for obtaining your private token](#how-to-generate-your-private-token). If you are still having trouble, here's some advice on some of the problems you might find:
+
+* If you receive a response code **401 Unauthorized**, this might mean that you are not providing your token correctly. Double check [here](#authentication) and make sure that you are sending the token in the *Authorization* header and correctly formatted.
+
+* A **401 Unauthorized** might also mean that you don't have the required apps associated to your user profile to access the application you are trying to access. Check the project where you want to perform requests (for instance, the Resource Watch API uses the `rw` application slug) and check if your user has the correct app. If you need to edit the apps of your user profile, check out the [PATCH auth/me](#update-your-user-account-details) endpoint.
+
+* A **403 Forbidden** usually means that you don't have the correct role for performing the operation you are trying to perform. Please confirm in the documentation of each endpoint the role required for performing the request.
```

Comment:
not just role. can also be apps, or other permission-related things. Generically speaking, a 403 is the API telling you "I know who you are, I believe you are who you say you are, you just don't have permissions to do what you're trying to do"

Reply:
👍 gonna broaden the forbidden text a bit then